diagrams/VerifierIssuerCollusion.drawio

```diff
@@ -0,0 +1,63 @@
+          <mxGeometry relative="1" as="geometry">
+            <mxPoint x="541" y="350" as="sourcePoint" />
+          </mxGeometry>
+        </mxCell>
+        <mxCell id="06Zk1V-yLQGSXVfKxCOx-4" value="&lt;span&gt;Verifier 2&lt;/span&gt;" style="ellipse;whiteSpace=wrap;html=1;rounded=0;shadow=0;comic=0;labelBackgroundColor=none;strokeWidth=1;fontFamily=Verdana;fontSize=12;align=center;fillColor=none;" parent="1" vertex="1">
+          <mxGeometry x="475" y="270" width="75" height="50" as="geometry" />
+        </mxCell>
+        <mxCell id="06Zk1V-yLQGSXVfKxCOx-8" value="Verifiable Presentation" style="edgeStyle=none;rounded=1;html=1;labelBackgroundColor=none;startArrow=none;startFill=0;startSize=5;endArrow=classicThin;endFill=1;endSize=5;jettySize=auto;orthogonalLoop=1;strokeColor=#B3B3B3;strokeWidth=1;fontFamily=Verdana;fontSize=12;exitX=1;exitY=0.5;exitDx=0;exitDy=0;" parent="1" source="1ea317790d2ca983-2" target="06Zk1V-yLQGSXVfKxCOx-9" edge="1">
+          <mxGeometry relative="1" as="geometry">
+            <mxPoint x="725" y="210" as="sourcePoint" />
+          </mxGeometry>
+        </mxCell>
+        <mxCell id="06Zk1V-yLQGSXVfKxCOx-9" value="&lt;span&gt;Verifier N&lt;/span&gt;" style="ellipse;whiteSpace=wrap;html=1;rounded=0;shadow=0;comic=0;labelBackgroundColor=none;strokeWidth=1;fontFamily=Verdana;fontSize=12;align=center;fillColor=none;" parent="1" vertex="1">
+          <mxGeometry x="690" y="240" width="75" height="50" as="geometry" />
+        </mxCell>
+        <mxCell id="cMt4WiRhtkRWSCWRY2kM-1" value="Colllusion&lt;br&gt;data sharing" style="endArrow=classic;startArrow=classic;html=1;rounded=0;exitX=0.347;exitY=0.078;exitDx=0;exitDy=0;dashed=1;strokeWidth=2;strokeColor=#FF0000;entryX=0;entryY=0.5;entryDx=0;entryDy=0;exitPerimeter=0;labelBackgroundColor=none;" parent="1" source="1ea317790d2ca983-7" target="1ea317790d2ca983-1" edge="1">
+          <mxGeometry width="50" height="50" relative="1" as="geometry">
+            <mxPoint x="490" y="240" as="sourcePoint" />
+            <mxPoint x="320" y="400" as="targetPoint" />
+            <Array as="points">
+              <mxPoint x="370" y="170" />
+            </Array>
+          </mxGeometry>
+        </mxCell>
+        <mxCell id="cMt4WiRhtkRWSCWRY2kM-2" value="Collusion&lt;br&gt;data sharing" style="endArrow=classic;startArrow=classic;html=1;rounded=0;exitX=0.999;exitY=0.227;exitDx=0;exitDy=0;entryX=0.651;entryY=-0.007;entryDx=0;entryDy=0;dashed=1;strokeWidth=2;strokeColor=#FF0000;entryPerimeter=0;exitPerimeter=0;labelBackgroundColor=none;" parent="1" source="1ea317790d2ca983-1" target="06Zk1V-yLQGSXVfKxCOx-9" edge="1">
```

Comment:
```suggestion
        <mxCell id="cMt4WiRhtkRWSCWRY2kM-2" value="Collusive&lt;br&gt;data sharing" style="endArrow=classic;startArrow=classic;html=1;rounded=0;exitX=0.999;exitY=0.227;exitDx=0;exitDy=0;entryX=0.651;entryY=-0.007;entryDx=0;entryDy=0;dashed=1;strokeWidth=2;strokeColor=#FF0000;entryPerimeter=0;exitPerimeter=0;labelBackgroundColor=none;" parent="1" source="1ea317790d2ca983-1" target="06Zk1V-yLQGSXVfKxCOx-9" edge="1">
```

Reply:
@TallTed I'm going to have to edit the diagrams in drawio and regenerate the SVGs.